<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemDialog</class>
 <widget class="QDialog" name="MemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MemDialog  {background-color: rgb(245,255,250)}

QLabel { color: black; }
QLineEdit { border-radius: 3px}

QPushButton
{
	color: black;
	font: 12pt &quot;Segoe UI&quot; bold;
	background-color: rgb(255,255,240);
	border-width: 2px;
    border-radius:3px;
}

QPushButton:hover {background-color: rgb(255,222,173)}

QLabel
{
	font: 12pt &quot;Segoe UI&quot; bold;
}

#membershipBox {
	background-color: rgb(255,250,250);
	border-radius: 8px;
}

#profileBox {
	background-color: rgb(255,250,250);
	border-radius: 8px;
}

#programBox {
	background-color: rgb(255,250,250);
	border-radius: 8px;
}
 </string>
  </property>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentDateTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI 13</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="membershipBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Membership</string>
   </property>
   <widget class="QLabel" name="membershipLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI 13</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Membership</string>
    </property>
   </widget>
   <widget class="QComboBox" name="membershipComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None Selected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bronze</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Silver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gold</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="membershipPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="programBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Programs</string>
   </property>
   <widget class="QComboBox" name="programComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None Selected</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Balance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Endurance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flexibility</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strength</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="programLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI 13</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Programs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="planPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="profileBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile</string>
   </property>
   <widget class="QPushButton" name="workoutPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Workout</string>
    </property>
   </widget>
   <widget class="QLabel" name="strengthLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="flexibilityLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="enduranceLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="faceLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
